app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -923,3 +927,34 @@ impl display::Object for Node {
         &self.model.display_object
     }
 }
+
+
+// === Positioning ===
+
+fn view_position_of_node_with_size(size: Vector2) -> Vector2 {
+    Vector2(size.x / 2.0, 0.0)
+}
+
+fn visualization_position_of_node_with_size(size: Vector2) -> Vector2 {
+    view_position_of_node_with_size(size) + Vector2(0.0, VISUALIZATION_OFFSET_Y)
+}
+
+fn bounding_box(
+    node_size: Vector2,
+    node_position: Vector2,
+    visualization_enabled_and_visible: bool,
+    visualization_size: Vector2,
+) -> BoundingBox {
+    let node_view_pos = node_position + view_position_of_node_with_size(node_size);
+    let node_bbox_pos = node_view_pos - node_size / 2.0;
+    let mut node_bbox = BoundingBox::from_position_and_size(node_bbox_pos, node_size);
+    if visualization_enabled_and_visible {
+        let visualization_pos_of_node = visualization_position_of_node_with_size(node_size);
```

Comment:
@MichaelMauderer 5 days ago, I got [a comment](https://github.com/enso-org/enso/pull/3301#discussion_r819310596) from @wdanilo that:

> "relative to position of node" does not carry any more information than just "of node".

I'm happy to set the name to whatever both of you agree on.

Reply:
I'm just thrown of by `of node`, as this does not seem quite grammatical. I would call it `visualization_position_relative_to_node`, or maybe shorter `visualization_offset`.

Reply:
For consistency with the name `background_offset` [proposed above](https://github.com/enso-org/enso/pull/3301#discussion_r822598260), I plan to go with `visualization_offset` here. In case of further disagreement, please continue the discussion on Discord so that we can quickly get to a resolution and I can use some names you both agree on.